samples/event-sourcing-esdb-simple/src/main/java/io/eventdriven/ecommerce/core/events/InMemoryEventBus.java

```diff
@@ -0,0 +1,30 @@
+package io.eventdriven.ecommerce.core.events;
+
+import io.eventdriven.ecommerce.core.scopes.ServiceScope;
+import org.springframework.core.ResolvableType;
+
+public record InMemoryEventBus(ServiceScope serviceScope) implements EventBus {
```

Comment:
If you want to show how to connect Event Sourcing with Spring framework ecosystem, I think better example would be to use Spring ApplicationEvents by mean of `ApplicationEventPublisher` and `@EventListener` annotation, which is commonly used in Spring.
You can find examples here:
- https://reflectoring.io/spring-boot-application-events-explained/
- [ApplicationEventPublisher](https://github.com/ddd-by-examples/library/blob/master/src/main/java/io/pillopl/library/commons/events/publisher/JustForwardDomainEventPublisher.java)

Reply:
Wow, I didn't know about that dummy me. Now it appears that I wrote a lot of useless code with my poor-mans event bus and service scope 😅 Puty, but I'll need to throw it away, lol. Thanks!

Reply:
Updated in https://github.com/oskardudycz/EventSourcing.JVM/pull/1/commits/949f0b4834ea0a4aab4b379f10b002d6ed7a81ab.